installer/install.iss

```diff
@@ -578,6 +572,66 @@ begin
     end;
 end;
 
+var
+    BuiltinFSMonitorStopOption,AlreadyHandledFSMonitorPaths:String;
+
+// Returns true if at least one FSMonitor daemon was shut down successfully
+function ShutdownFSMonitorDaemons():Boolean;
+var
+    FindRec:TFindRec;
+    Path,Str:String;
+    Len,i:Integer;
+begin
+    Result:=False;
+#ifdef WITH_EXPERIMENTAL_BUILTIN_FSMONITOR
+    if (BuiltinFSMonitorStopOption='(huh?)') then
+        Exit;
+    if not FindFirst('\\.\pipe\*',FindRec) then
+        Exit;
+    if (AlreadyHandledFSMonitorPaths='') then
+        AlreadyHandledFSMonitorPaths:=#0;
+    repeat
+        if WildcardMatch(FindRec.Name,'*\fsmonitor--daemon.ipc') then begin
+            if (Pos(#0+FindRec.Name+#0,AlreadyHandledFSMonitorPaths)>0) then
+                Continue;
+            AlreadyHandledFSMonitorPaths:=AlreadyHandledFSMonitorPaths+FindRec.Name+#0;
+
+            // An earlier `fsmonitor--daemon` iteration called it `--stop`, not `stop`;
+            // Find out which form to use.
+            if (BuiltinFSMonitorStopOption='') then begin
+                BuiltinFSMonitorStopOption:='(huh?)';
+                Path:=ExpandConstant('{tmp}\fsmonitor--help.err');
+                if not Exec(ExpandConstant('{sys}\cmd.exe'),'/D /C ""'+AppDir+'\cmd\git.exe" fsmonitor--daemon -h 2>"'+Path+'""','',SW_HIDE,ewWaitUntilTerminated,i) or (i<>129) then begin
```

Comment:
Hmm. I guess we should be quiet about this (and exit early) if

- `AppDir+'\cmd\git.exe'` does not even exist (exit code 127), or
- FSMonitor was not included in the previously-installed Git for Windows version (i.e. if `git fsmonitor--daemon -h` returns exit code 1.

My current thinking is that the error message should be suppressed if the exit code (i.e. `i`) is either 1 or 127.

Reply:
Done.